internal/aez/reader.go

```diff
@@ -0,0 +1,67 @@
+package aez
+
+import (
+	"bytes"
+	"encoding/binary"
+	"log"
+
+	"github.com/innosat-mats/rac-extract-payload/internal/innosat"
+)
+
+//PackageType ...
+type PackageType interface {
+}
+
+// Packages ...
+func Packages(source <-chan innosat.SourcePackage, target chan<- PackageType) {
```

Comment:
cant we replace the copypaste inside the inner switch with something like:
`data = readOrLog(structType interface{Read})` 

and if that is possible we could even have a `(sid)map[interface{Read}])` or something and remove the entire switch.

if not consider at least breaking apart into several functions, we not have a for -> if -> if -> switch

missing tests for:
* GroupingFlags not standalone
* TC header type
* Not main application
* HTR, PWR, CPRUA and CPRUB sids / packages. Both valid and bad versions.